meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,121 @@
+# Wayfair plugin: Initial Setup

```

Comment:
We should also include documentation specific to upgrading plugin versions.

Reply:
I was about to tell you I was adding that!

Reply:
@mcloughlin77 see the new `updating.md` file, which is now referenced from the main user guide.